ipatests/test_integration/test_acme.py

```diff
@@ -93,6 +131,9 @@ def test_acme_service_not_yet_enabled(self):
             ['curl', '--fail', self.acme_server],
             ok_returncode=22,
         )
+        result = self.master.run_command(['ipa-acme-manage', 'status'])
+        assert result.returncode == 0
```

Comment:
This check can be removed., the return code for status command will be `0` anyways.

Reply:
Yes, I suppose any exception is sufficient in case something goes wrong.